### PR TITLE
Use apollo useQuery hook for Compound subgraph data for savings

### PR DESCRIPTION
**Pull request by jinchung**
Moved to use the Apollo client useQuery hook to fetch Compound subgraph data for savings. 

Previously we were having issues where the subscription would not properly finish when switching wallets and we would get data overwritten from other wallets.

### CODE REVIEW

**Comment by jinchung** [comment]
At the moment, the Compound v2 Subgraph is having an issue with one of their triggers, so the data is out of sync.

File: src/hooks/useSavingsAccount.js [hunk 1]
```diff
@@ -1,49 +1,198 @@
...
+        savingsAccountData = orderBy(
+          parsedAccountTokens,
+          ['ethPrice'],
+          ['desc']
+        );
+      } else if (loading) {
+        savingsAccountData = backupAccountTokens;
+      }
+
+      saveSavings(savingsAccountData, accountAddress, network);
```
Comment (brunobar79) [inline src/hooks/useSavingsAccount.js:153]: This should me moved inside the if cause there's no reason to read and write the same data to localstorage

**Review by brunobar79 — APPROVED** [review]
1 minor comment but LGTM

**Comment by brunobar79** [comment]
@jin you might wanna look at why `parseSavingsResult` gets called 3 times in a row every time the polling happens.

**Comment by jinchung** [comment]
> 1 minor comment but LGTM

Great catch! Thanks!

**Review by skylarbarrera — APPROVED** [review]
🙆‍♂️ , not seeing the issue anymore, lmk if you want to me check again after you address comment

**Comment by jinchung** [comment]
> @jin you might wanna look at why `parseSavingsResult` gets called 3 times in a row every time the polling happens.

Nice catch! I'm going to address this in a separate PR. It isn't making a network request multiple times but it is doing parsing multiple times.

**Pull request merged**